gsa/src/web/components/dashboard/dashboard.js

```diff
@@ -61,7 +61,11 @@ import withGmp from 'web/utils/withGmp';
 import compose from 'web/utils/compose';
 
 import {getDisplay} from './registry';
-import {getRows} from './utils';
+import {getRows as get_rows} from './utils';
```

Comment:
Does this mean, "camelCase API" is not a thing anymore?

Reply:
snake_case should not be a thing anymore! All our code use camelCase now. All old code needs to be converted to use camelCase.

Reply:
Why the ... as ... import than? Is this necessary at this place? I mean, even in the old code we have one location where we will always find snake_case, namely in the commands...

Reply:
It's necessary because we have a method called getRows that calls getRows and react returns an error. It wasn't a problem in a class component. Not sure why, but I suspect it's because in class components it's technically "this.getRows". I can name it something else, but right now it looks like it has to be named differently from the internal dashboard function getRows.

Reply:
Then it should be named differently without snake_case. I find it suboptimal, though, that we seemingly had two getRows in one component - one being a function in utils and one being a class method calling this function. Their naming should have been a little more precise from the beginning on.

Reply:
Just saw that you already made changes :) Thanks!